### PR TITLE
Add feature extraction

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Add feature extraction

In this PR, we perform segmentation and feature extraction using CellProfiler and save morphology features as an SQLite file (not included due to size).

### CODE REVIEW

File: 3.feature_extraction/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Perform segmentation and extract morphology features
+
+In this module, we use CellProfiler to segment nuclei, cells, cytoplasm, and organoids, also known as compartments.
+We then extract morphology features (e.g., texture, granularity, area, etc.) using the compartments as masks across all channels.
+We also extract some whole image features, but these are not prioritized in downstream analysis.
```
Comment (gwaybio) [inline 3.feature_extraction/README.md:5]: what do you mean "prioritized"? Do you actually mean not considered?

FWIW, in this work, whole image might actually be interesting.... oh, but wait, you're performing whole organoid feature extraction so it's probably very similar

File: 3.feature_extraction/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+We are focused on extracting `single-cell` (nuclei, cells, and cytoplasm) and `whole organoid` (organoids) features.
+
+It took approximately **~5 hours** to segment and extract features into an SQLite database.
```
Comment (gwaybio) [inline 3.feature_extraction/README.md:8]: how many images?

**Review by gwaybio — APPROVED** [review]

File: 3.feature_extraction/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Perform segmentation and extract morphology features
+
+In this module, we use CellProfiler to segment nuclei, cells, cytoplasm, and organoids, also known as compartments.
```
[the remark this answers is not in the document]
Comment (jenna-tomkinson) [reply 3.feature_extraction/README.md:3]: I have corrected this to be more clear!

File: 3.feature_extraction/README.md (hunk 1 above)
Comment (jenna-tomkinson) [reply 3.feature_extraction/README.md:5]: Maybe we can say whole image features are not included in downstream analysis which is true for all projects, plus I agree whole organoid would probably be very similar to whole images anyway.

File: 3.feature_extraction/README.md (hunk 2 above)
Comment (jenna-tomkinson) [reply 3.feature_extraction/README.md:8]: Added!

**Comment by jenna-tomkinson** [comment]
@gwaybio Thank you for the review! I am merging now!

**Pull request merged**